src/TeleopMode.cpp

```diff
@@ -98,20 +106,39 @@ void Teleop::TeleopPeriodic() {
                 m_intakeAssembly->HardCloseClaw();
                 m_intakeAssembly->Flash();
                 m_intakeModeTimer = GetMsecTime();
+                m_operatorJoystick->SetRumble(
+                    GenericHID::RumbleType::kLeftRumble, 1);
+                m_operatorJoystick->SetRumble(
+                    GenericHID::RumbleType::kRightRumble, 1);
                 m_cubeIntakeState = CubeIntakeState::SwitchIntakeDelay;
             }
             break;
         case CubeIntakeState::SwitchIntakeDelay:
             if (GetMsecTime() - m_intakeModeTimer > 100) {
                 m_intakeAssembly->GoToIntakePosition(
                     IntakeAssembly::STOW_PRESET);
+                m_intakeModeTimer = GetMsecTime();
+                m_cubeIntakeState = CubeIntakeState::StopRumble;
+            }
+            break;
+        case CubeIntakeState::StopRumble:
+            if (GetMsecTime() - m_intakeModeTimer > 500) {
+                m_operatorJoystick->SetRumble(
+                    GenericHID::RumbleType::kLeftRumble, 0);
+                m_operatorJoystick->SetRumble(
+                    GenericHID::RumbleType::kRightRumble, 0);
                 m_cubeIntakeState = CubeIntakeState::Idle;
             }
             break;
         case CubeIntakeState::ManualIntaking:
             m_intakeAssembly->RunIntake(-1.0);
             if (m_intakeAssembly->GetClaw()->IsCubeIn()) {
-                m_cubeIntakeState = CubeIntakeState::Idle;
+                m_operatorJoystick->SetRumble(
+                    GenericHID::RumbleType::kLeftRumble, 1);
+                m_operatorJoystick->SetRumble(
```

Comment:
This is dope

Reply:
Rumble and all joysticks worked pretty fine! We should be using this at MTTD.